framework/src/modules/chain/helpers/z_schema.js

```diff
@@ -14,411 +14,17 @@
 
 'use strict';
```

Comment:
Please open add a comment that z-schema will be deprecated after https://github.com/LiskHQ/lisk/issues/2546

Reply:
We can still work on issue #2546 to completely remove z-schema from chain module.